extensions/amp-consent/cmps/sirdata.md

```diff
@@ -0,0 +1,48 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
nit: 2020 : )

Reply:
Complete your call. Since you've linked to your own doc in the `amp-consent.md`. It's fine to not include this doc to AMP repo.